packages/field/src/react/field.tsx

```diff
@@ -51,7 +53,7 @@ interface FieldProps extends Omit<HTMLPropsFor<'div'>, 'prefix' | 'ref'> {
   label?: ReactNode
   onClick?: MouseEventHandler<HTMLDivElement>
   prefix?: ReactNode
-  renderContainer?: ForwardRefExoticComponent<unknown>
+  renderContainer?: ForwardRefExoticComponent<RefAttributes<any>>
```

Comment:
why the switch?

Reply:
i was getting errors trying to apply the prop because the RefAttribute was necessary - i feel like i'm still learning how all these types work

Reply:
srsly